themes/ddbasic/sass/module/_ding_nodelist.scss

```diff
@@ -0,0 +1,1101 @@
+  border-color: #ccc #c1c1c1 #c1c1c1 #ccc;
+  border-style: solid;
+  border-width: 1px;
+  -webkit-box-shadow: 3px 3px 4px #c9c9c9;
+  -moz-box-shadow: 3px 3px 4px #c9c9c9;
+  box-shadow: 3px 3px 4px #c9c9c9;
+}
+
+.event_image .field-items {
+  height: 180px;
+}
+
+.news_image {
+  float:left;
+  margin-right: 20px;
+  border-color: #ccc #c1c1c1 #c1c1c1 #ccc;
+  border-style: solid;
+  border-width: 1px;
+  -webkit-box-shadow: 3px 3px 4px #C9C9C9;
+  -moz-box-shadow: 3px 3px 4px #C9C9C9;
+  box-shadow: 3px 3px 4px #C9C9C9;
+}
+
+.artile_image .field-item {
+  width: 220px;
+  height: 160px;
+}
+
+/* Name and date
+   ========================================================================== */
+
+.bottom {
+  padding-bottom: 10px;
+  clear: both;
+
+  ul {
+    padding: 0;
+    margin: 0;
+
+    li {
+      float: left;
+      list-style: none;
+      margin-right: 20px;
+      color: #6A6960;
+      height: 20px;
+    }
+  }
+}
+
+.name {
+  background: url(../images/node-author.png) no-repeat left bottom;
+  display: block;
+  height: 20px;
+  padding-left: 15px;
+}
+
+.date {
+  float: right;
+  background: url(../images/node-created.png) no-repeat left bottom;
+  display: block;
+  height: 20px;
+  padding-left: 19px;
+}
+
+/* More links
+   ========================================================================== */
+
+.ding_nodelist-horizontal_accordeon {
+  li.slide-closed .data {
+    display: none;
+  }
+
+  li.slide-open .data {
+    display: block;
+  }
+
+  li {
+    position: relative;
+    height: 350px;
+
+    .event_date {
+      position: absolute;
+    }
+  }
+}
+
+/* Slider base styles
+   ========================================================================== */
+
+.ding_nodelist-slider {
+  height: 400px;
+  overflow: hidden;
+
+  .legend {
+    float: left;
+    width: 300px;
+  }
+
+  .ding_nodelist-items {
+    float: left;
+    width: 600px;
+
+    .item {
+      width: 100%;
+    }
+  }
+}
+
+.ding_nodelist-vertical_accordion .va-slice {
+  cursor: pointer;
+}
+
+
+// Variables
+//==========================================================================
+
+$label-bg: url('../images/p.png');
+
+
+// All nodelists
+//==========================================================================
+
+.col3 .ding_nodelist-taxonomy .item .item-list-image,
+.col4 .ding_nodelist-taxonomy .item .item-list-image {
+  display: none;
+}
+
+.ding_nodelist {
+  overflow: hidden;
+  margin-bottom: 20px;
+
+  .node .item-date,
+  .node .library,
+  .node .item-price,
+  .data .event-timestamp,
+  .data .event-details,
+  .item-details .item-date,
+  .item-details .item-library,
+  .item-details .item-price {
+    font-size: 80%;
+  }
+
+  .node > p {
+    margin: 10px 0;
+  }
+
+  .node-title {
+    text-decoration: underline;
+
+    a {
+      color: $white;
+    }
+  }
+
+  .more {
+    display: none;
+  }
+
+  // Event timestamp
+  .event-time {
+    position: absolute;
+    top: 10px;
+    left: 10px;
+    z-index: 49;
```

Comment:
Please add a comment explaining what the z-index chosen is in releation to. In other words: Why 49?.